lru_cache/lru_cache.py

```diff
@@ -30,4 +35,15 @@ def get(self, key):
     the newly-specified value.
     """
     def set(self, key, value):
-        pass
+        d_value = [key, value]
+        if not self.storage.get(key, False):
```

Comment:
Nice use of `get()` here again, but the default return for a key that's not found is `false` by the way.